bindings/pyroot_experimental/pythonizations/python/ROOT/_numbadeclare.py

```diff
@@ -0,0 +1,309 @@
+from cppyy import gbl as gbl_namespace
```

Comment:
We can leave this file here, for now. But perhaps at some point we will need to reorganize. Everything that is not a pythonization but belongs more to the interface of the ROOT module, could go in an e.g. `interface` submodule (containing now `_facade.py`, `_numbadeclare.py`, `_application.py`).